waspc/e2e-test/test-outputs/waspBuild-golden/files.manifest

```diff
@@ -1,4 +1,5 @@
 waspBuild/.wasp/build/.dockerignore
+waspBuild/.wasp/build/.waspchecksums
```

Comment:
aha, so we have checksums in build/ folder too, when a user runs `wasp build`? I guess this way we avoid extra work during the build phase as well?

Reply:
Ah, e2e manifests still helping already :D yeah we just used the `outDir` and did not differentiate/care which out dir it was.